### ISSUE TITLE
System test generator appears to ignore --spec option

### ISSUE THREAD

**Issue by bobmazanec** [minitest/minitest-rails#253]
because its template file, [`lib/generators/minitest/system/templates/system_spec.rb.tt`](https://github.com/minitest/minitest-rails/blob/v7.0.0/lib/generators/minitest/system/templates/system_spec.rb.tt),
is still in the `Test` style

```
require "application_system_test_case"

class <%= class_name.pluralize %>Test < ApplicationSystemTestCase
  # test "visiting the index" do
  #   visit <%= plural_table_name %>_url
  #
  #   assert_selector "h1", text: "<%= class_name %>"
  # end
end
```

**Comment by blowmage** [comment]
Oh wild, you are right. It should match the [scaffold template](https://github.com/minitest/minitest-rails/blob/v7.0.0/lib/generators/minitest/scaffold/templates/system_spec.rb.tt), but doesn't. I'll fix this.

**Comment by blowmage** [comment]
This is included in [minitest-rails 7.1.0](https://rubygems.org/gems/minitest-rails/versions/7.1.0) and [minitest-rails 7.0.1](https://rubygems.org/gems/minitest-rails/versions/7.0.1). Are you using a version older than that?

**Comment by bobmazanec** [comment]
[7.0.0](https://rubygems.org/gems/minitest-rails/versions/7.0.0), but looking forward to moving to both minitest-rails & rails 7.1.0 😄

Thanks!

**Comment by blowmage** [comment]
Excellent. Thanks for your support!

I went ahead and released [minitest-rails 6.1.1](https://rubygems.org/gems/minitest-rails/versions/6.1.1) and [minitest-rails 6.0.2](https://rubygems.org/gems/minitest-rails/versions/6.0.2) with this fix while I was at it...
